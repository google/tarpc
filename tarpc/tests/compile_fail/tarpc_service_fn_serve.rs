#[tarpc::service]
trait World {
    async fn serve();
}

fn main() {}
