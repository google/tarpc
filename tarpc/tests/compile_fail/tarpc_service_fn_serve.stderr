error: method name conflicts with generated fn `World::serve`
 --> $DIR/tarpc_service_fn_serve.rs:3:14
  |
3 |     async fn serve();
  |              ^^^^^
